.github/workflows/release.yml

```
@@ -0,0 +1,33 @@
+          cache: "gradle"
+
+      - name: Build MineVNLib
+        run: |
+          chmod +x ./gradlew
+          ./gradlew assemble
+          mkdir -p target
+          cp ./minevnlib-plugin/build/libs/MineVNLib.jar ./target/MineVNLib.jar
+          cp ./minevnlib-plugin/build/libs/MineVNLib-no-kotlin.jar ./target/MineVNLib-no-kotlin.jar
+
+      - name: Attach JARs to the Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)